meals/templates/meal_add.html

```diff
@@ -0,0 +1,69 @@
+{% extends "base_tabs.html" %}
+{% block head_title %}Add meal{% endblock %}
+{% block cols-panel %}col-md-6 col-md-offset-3{% endblock %}
+{% block out_panel %}
+    <form id="meal-form" action="" method="post">
+	{% csrf_token %}
+	<div class="form-group">
+	    <label class="control-label" for="meal_name">Name</label>
+	    <input type="text" class="form-control" name="meal_name" />
+	</div>
+	<div class="form-group">
+	    <label class="control-label" for="meal_name">Type</label>
+	    <select class="form-control" name="meal_type">
+	    {% for type in types %}
+	        <option value="{{ type.0 }}">{{ type.1 }}</option>
+	    {% endfor %}
+	    </select>
+	</div>
+	<div class="form-group">
+	    <label class="control-label" for="meal_starts">Starting time</label>
+	    <input type="text" class="form-control" name="meal_starts" value="{{ time1 }}" />
+	</div>
+	<div class="form-group">
+	    <label class="control-label" for="meal_ends">Ending time</label>
+	    <input type="text" class="form-control" name="meal_ends" value="{{ time2 }}" />
+	</div>
+	<div class="form-group">
+	    <label class="control-label" for="meal_times">Times</label>
```

Comment:
Change to "Number of times a hacker can eat" or put that as help text pls.

Reply:
On #52.

Reply:
Changed.